projects/js-packages/publicize-components/src/social-store/types.ts

```diff
@@ -32,10 +32,8 @@ export type JetpackSettings = {
 	showNudge?: boolean;
 };
 
-export type ShareStatusItem = Pick<
-	Connection,
-	'connection_id' | 'profile_link' | 'profile_picture'
-> & {
+export type ShareStatusItem = Pick< Connection, 'profile_link' | 'profile_picture' > & {
+	connection_id: number;
```

Comment:
One of the reasons I had advocated for having the same schema for both for share status and connections.

For share status we have `connection_id` as integer but for connections it is a numeric string. 🤷